certified-connectors/Almabase/apiDefinition.swagger.json

```diff
@@ -1,4779 +1,7275 @@
+                  "type": "string",
+                  "x-ms-summary": "Payment External Database ID"
+              },
+              "phone_number": {
+                  "maxLength": 64,
+                  "type": "string",
+                  "x-ms-summary": "Phone Number"
+              },
+              "refund_amount": {
+                  "type": "string",
+                  "x-ms-summary": "Refund Amount"
+              },
+              "registration_status": {
+                  "type": "string",
+                  "x-ms-enum": [
+                      "pending_payment",
+                      "registration_successful",
+                      "registration_cancelled"
+                  ],
+                  "x-ms-summary": "Registration Status"
+              },
+              "registration_status_display": {
+                  "type": "string",
+                  "x-ms-summary": "Registration Status Display"
+              },
+              "rsvp_status": {
+                  "type": "string",
+                  "x-ms-enum": [
+                      "yes",
+                      "no",
+                      "maybe"
+                  ],
+                  "x-ms-summary": "RSVP status"
+              },
+              "rsvp_status_display": {
+                  "type": "string",
+                  "x-ms-summary": "RSVP status display"
+              },
+              "tickets": {
+                  "items": {
+                      "$ref": "#/definitions/v1.EventsApi.TicketBoughtRead"
+                  },
+                  "type": "array"
+              },
+              "total_amount": {
+                  "type": "string",
+                  "x-ms-summary": "Total Amount"
+              },
+              "transactions": {
+                  "items": {
+                      "$ref": "#/definitions/v1.EventsApi.TransactionRead"
+                  },
+                  "type": "array"
+              },
+              "utm_campaign": {
+                  "type": "string",
+                  "x-ms-summary": "UTM campaign"
+              },
+              "utm_medium": {
+                  "type": "string",
+                  "x-ms-summary": "UTM Medium"
+              },
+              "utm_source": {
+                  "type": "string",
+                  "x-ms-summary": "UTM Source"
+              },
+              "venue_location": {
+                  "type": "string",
+                  "x-ms-summary": "Venue Location"
+              },
+              "zoom_registration": {
+                  "properties": {
+                      "registrant_id": {
+                          "type": "string",
+                          "x-ms-summary": "Zoom Registrant ID"
+                      },
+                      "response": {
+                          "type": "object",
+                          "x-ms-summary": "Response from zoom"
+                      }
+                  },
+                  "type": "obj
```

Comment:
@ab-sahira make sure that everything is grammatically correct and is free of typographical errors. You refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).